File: sample/src/main/java/com/u1f4f1/sample/HoursModel.java
package com.u1f4f1.sample;

import android.view.View;

import com.airbnb.epoxy.EpoxyAttribute;
import com.airbnb.epoxy.EpoxyModel;
import com.airbnb.epoxy.EpoxyModelClass;

@EpoxyModelClass(layout = R.layout.model_business_hours)
public abstract class HoursModel extends EpoxyModel<HoursView> {
    @EpoxyAttribute(hash = false) public Runnable buttonClickConsumer;

    @Override
    public void bind(final HoursView view) {
        view.hoursContainer.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                buttonClickConsumer.run();

                view.cardClicked();
            }
        });
    }
}

File: sample/src/main/java/com/u1f4f1/sample/HoursModel.kt
package com.u1f4f1.sample

import com.airbnb.epoxy.EpoxyAttribute
import com.airbnb.epoxy.EpoxyModel
import com.airbnb.epoxy.EpoxyModelClass

@EpoxyModelClass(layout = R.layout.model_business_hours)
abstract class HoursModel : EpoxyModel<HoursView>() {
    @EpoxyAttribute(hash = false)
    var buttonClickConsumer: Runnable? = null

    override fun bind(view: HoursView?) {
        view?.hoursContainer?.setOnClickListener({
            buttonClickConsumer!!.run()

            view.cardClicked()
        })
    }
}
